simtools/applications/generate_default_metadata.py

```diff
@@ -83,14 +85,25 @@ def main():
     if args_dict["output_file"] is None:
         print(default_values)
     else:
-        _logger.info(f"Writing default values to {args_dict['output_file']}")
-        with open(args_dict["output_file"], "w", encoding="utf-8") as file:
-            yaml.dump(
-                default_values,
-                file,
-                default_flow_style=False,
-                sort_keys=False,
-            )
+        _io_handler = io_handler.IOHandler()
+        _out_file = _io_handler.get_output_file(args_dict["output_file"])
+        _logger.info(f"Writing default values to {_out_file}")
+        if args_dict["output_file"].endswith((".yml", ".yaml")):
```

Comment:
I would rather use `_out_file` and not `args_dict["output_file"]` in these two next if statements. It seems safer to me, as  `_out_file` already passed through the io_handler.

Reply:
This is checking for the file suffix only, so we can use directly the args_dict value. io_handler is doing the work on the full path, which is not needed here.

Reply:
ok.